CONTRIBUTING.md

```diff
@@ -31,132 +33,166 @@ There are many ways in which you can contribute, beyond writing code. The goal o
 
 ### Look For an Existing Issue
+
 Before you create a new issue, please do a search in [open issues](https://github.com/xfox111/TabsAsideExtension/issues) to see if the issue or feature request has already been filed.
 
 Be sure to scan through the [feature requests](https://github.com/XFox111/TabsAsideExtension/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement).
 
 If you find your issue already exists, make relevant comments and add your [reaction](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments). Use a reaction in place of a "+1" comment:
 
-* 👍 - upvote
-* 👎 - downvote
+- 👍 - upvote
+- 👎 - downvote
 
 If you cannot find an existing issue that describes your bug or feature, create a new issue using the guidelines below.
 
 ### Writing Good Bug Reports and Feature Requests
+
 File a single issue per problem and feature request. Do not enumerate multiple bugs or feature requests in the same issue.
 
 Do not add your issue as a comment to an existing issue unless they are the same ones. Many issues look similar, but have different causes.
 
 The more information you can provide, the more likely someone will be successful at reproducing the issue and finding a solution.
 
 Please include the following with each issue:
+
 - Current version of the extension
 - Your current browser and OS name
 - Reproducible steps (1... 2... 3...) that cause the issue
 - What you expected to see, versus what you actually saw
 - Images, animations, or a link to a video showing the issue occurring
 
 ### Final Checklist
+
 Please remember to do the following:
+
 - [ ] Search the issue repository to ensure your report is a new issue
 - [ ] Separate issues reports
 - [ ] Include as much information as you can to your report
 
 Don't feel bad if the developers can't reproduce the issue right away. They will simply ask for more information!
 
 ### Follow Your Issue
+
 Once your report is submitted, be sure to stay in touch with the devs in case they need more help from you.
 
-## Contributing to codebase
-If you are interested in writing code to fix issues or implement new awesome features you can follow this guidelines to get a better result
+## Contributing to the codebase
+
+If you are interested in writing code to fix issues or implement new awesome features you can follow these guidelines to get a better result
 
 ### Deploy test version on your browser
+
 1. Clone repository to local storage using [Git](https://guides.github.com/introduction/git-handbook/)
-    ```
-    git clone https://github.com/xfox111/TabsAsideExtension.git
-    ```
+
+   ```bash
+   git clone https://github.com/xfox111/TabsAsideExtension.git
+   ```
+
 2. Enable Developers mode on your browser extensions page
-3. Click "Load unpacked" button and navigate to the extension root folder (contains `manifest.json`)
+3. Click the "Load unpacked" button and navigate to the extension root folder (contains `manifest.json`)
```

Comment:
I was insecure too, Grammarly says that the article is necessary (screenshot below) Should I revert this change?

![image](https://user-images.githubusercontent.com/53786619/95768564-d611bd00-0cb6-11eb-9216-2340c21492c0.png)

Reply:
> It seems that article use may be incorrect here

Yeah, I guess Grammarly has its doubts as well :) But I still think that there's enough context to understand which exactly button we meant. So, yeah, it would be better to revert

Reply:
Ok, I'll revert that